ygot/render_test.go

```diff
@@ -1988,6 +1988,22 @@ func TestConstructJSON(t *testing.T) {
 			},
 			List: map[uint32]*renderExampleList{},
 		},
+		wantIETF: map[string]interface{}{
+			"ch": map[string]interface{}{"val": "42"},
+			// RFC7951 Section 5.4, the list is a name /array pair, an array must be [], not null RFC8259 Sectiob 5
```

Comment:
Perhaps this comment should read:

```
/// RFC7951 Section 5.4 defines a YANG list as an array when empty, the array should be [] rather than 'null' per RFC 8259 Section 5
```

At a minimum please fix the typo here (`s/Sectiob/Section/`).

Reply:
Comment now reads:
RFC7951 Section 5.4 defines a YANG list as an JSON array. Per RFC 8259 Section 5 an empty array should be [] rather than 'null'.